test/specs/anInitialSuite.e2e.ts

```diff
@@ -25,25 +25,26 @@ suite does run, it needs to run first.
 describe('An Initial Suite', async () => {
   let testSetup: TestSetup;
 
+  step('Countdown', async () => {
+    utilities.log('About to start the e2e tests...');
+    for (let i = 10; i > 0; i--) {
+      utilities.log(`${i}...`);
+      await utilities.pause(1);
+    }
+  });
+
   step('Verify our extensions are not initially loaded', async () => {
     const workbench = await browser.getWorkbench();
-    await utilities.runCommandFromCommandPrompt(
-      workbench,
-      'Developer: Show Running Extensions',
-      2
-    );
+    await utilities.showRunningExtensions(workbench);
```

Comment:
There wee a few places where we were testing/looking for running extension.  I moved the code into `showRunningExtensions()`, which also closes the output panel before looking for the extension, and the shows the output panel again.